sourcemash/frontend/static/js/views/register_modal_view.js

```diff
@@ -0,0 +1,41 @@
+Sourcemash.Views.RegisterModalView = Backbone.View.extend({
+    template: JST['register-modal'],
+
+    events: {
+        'submit #register': 'registerFromModal',
+    },
+
+    registerFromModal: function(e) {
+        e.preventDefault();
+        var formData = JSON.stringify($("#register").serializeObject());
+        $('#register-password').val('');
+        $('#register-password-confirm').val('');
+        var posting = $.ajax({
+                          type: "POST",
+                          url: "/register",
+                          data: formData,
+                          success: this.registerOrShowErrors,
```

Comment:
http://api.jquery.com/jquery.ajax/ shows that there is an 'error' option for an ajax call. It would probably make more sense to split the success and error callbacks if possible.